docs/source/additional/release.rst

```diff
@@ -31,6 +31,26 @@ This is an overview of the main changes, please have a look at the changelog of
 - `backend changelog <https://github.com/Substra/substra-backend/blob/main/CHANGELOG.md>`__
 - `orchestrator changelog <https://github.com/Substra/orchestrator/blob/main/CHANGELOG.md>`__
 
+Substra 0.35.0 --- 2024-02-20
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Platform** 
+
+- Remove distributed mode. Starting from this release, only the standalone mode is supported.
+
+**All**
+
```

Comment:
- Remove distributed orchestration mode based on the private blockchain. Starting from this release, only the standalone orchestration mode is supported.